src/components/ForecastSummary.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import WeatherIcon from "react-icons-weather";
+
+import PropTypes from "prop-types";
+
+function ForecastSummary(props) {
+  const { date, temperature, description, icon, setSelectedDate } = props;
```

Comment:
You could destructure props directly in the component definition and remove the destructuring from `props` on the next line:
```
function LocationDetails({ date, temperature, description, icon, setSelectedDate }) {
```